lib/node.js

```diff
@@ -1093,8 +1093,31 @@ declare module "util" {
   declare function deprecate(f: Function, string: string): Function;
 }
 
+type vm$ScriptOptions = {
+  filename?: string,
+  lineOffset?: number,
+  columnOffset?: number,
+  displayErrors?: boolean,
+  timeout?: number
+};
+
+declare class vm$Script {
+  constructor(code: string, options: Object): void;
+  runInContext(contextifiedSandbox: vm$Context, options?: vm$ScriptOptions): void;
+  runInNewContext(sandbox?: Object, options?: vm$ScriptOptions): void;
+  runInThisContext(options?: vm$ScriptOptions): void;
```

Comment:
It seems like the return value for `runIn*` methods should by `any`, like the `runIn*` functions below. We don't know what the code will eval to, but I'm pretty sure it's returned from this API.